website/README.md

```diff
@@ -1,43 +1,38 @@
-This website was created with [Docusaurus](https://docusaurus.io/).
-
 # What's In This Document
 
+This website was created with [Docusaurus](https://docusaurus.io/).
+
 * [Get Started in 5 Minutes](#get-started-in-5-minutes)
 * [Directory Structure](#directory-structure)
 * [Editing Content](#editing-content)
 * [Adding Content](#adding-content)
 * [Full Documentation](#full-documentation)
 
-# Get Started in 5 Minutes
+## Get Started in 5 Minutes
 
 1. Make sure all the dependencies for the website are installed:
 
 ```sh
-# Install dependencies
-$ npm install
+npm install
 ```
 
-2. Run your dev server:
+1. Run your dev server:
```

Comment:
"Make sure all the dependencies for the website are installed" is .1 - I think this should still be .2